packages/nimble-components/src/table/tests/table.stories.ts

```diff
@@ -127,7 +128,7 @@ const metadata: Meta<TableArgs> = {
             id-field-name="${x => dataSetIdFieldNames[x.data]}"
             data-unused="${x => x.updateData(x)}"
         >
-            <nimble-table-column-text field-name="firstName" placeholder="no value" column-id="first-name-column" action-menu-slot="name-menu" action-menu-label="Configure name">
+            <nimble-table-column-text sort-direction="ascending" sort-index="0" field-name="firstName" placeholder="no value" column-id="first-name-column" action-menu-slot="name-menu" action-menu-label="Configure name">
```

Comment:
I've updated the story to have an array of column-id/sort-direction pairs that is used to populate the `sort-direction` and `sort-index` attributes on each column.

Reply:
I think we should add Description text for the `sortedColumns` to let users know what the keys and values are for that editor. It might also be worth mentioning first, that `sortedColumns` isn't an API, but just a way of encapsulating the `sort-direction` and `sort-index` APIs for each column in a Storybook editor.

Reply:
done